app/parse.py

```diff
@@ -13,11 +43,127 @@ class Product:
     price: float
     rating: int
     num_of_reviews: int
+    additional_info: dict
+
+
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def close_cookie_banner():
+    driver = get_driver()
+    try:
+        WebDriverWait(driver, 10).until(
+            EC.presence_of_element_located((By.ID, "cookieBanner"))
+        )
+        cookie_button = driver.find_element(
+            By.ID, "cookieBanner"
+        ).find_element(By.TAG_NAME, "button"
+                       )
+
+        WebDriverWait(driver, 10).until(
+            EC.element_to_be_clickable((By.ID, "cookieBanner"))
+        )
+
+        driver.execute_script("arguments[0].click();", cookie_button)
+        print("Cookie banner closed successfully.")
+    except (NoSuchElementException, ElementNotInteractableException) as e:
+        print("Cookie banner not interactable or not present:", e)
```

Comment:
The exception handling in `close_cookie_banner()` only catches `NoSuchElementException` and `ElementNotInteractableException`. Consider adding a more general exception handling or logging to cover unexpected issues that might occur during interaction with the cookie banner.